### DISCUSSION TITLE
Dangerous site warning

### DISCUSSION THREAD

**Discussion by EncryptMan** [nextauthjs/next-auth#9961] [Help]
Sometimes after logging with Discord oauth I get "Dangerous Site" warning by chrome on my production website. I am using next js 14 and next-auth ^4.24.5. Hosting provider is Vercel with free plan. I don't know how to fix it and what is causing problem.

<img width="544" alt="Screenshot 2024-02-08 at 19 34 07" src="https://github.com/nextauthjs/next-auth/assets/136573652/09d8fc82-a704-459c-a11e-f8832a7a6871">
<img width="1383" alt="Screenshot 2024-02-08 at 19 34 31" src="https://github.com/nextauthjs/next-auth/assets/136573652/80d815cf-5a80-4a48-b1a1-c7403cd53a68">

**Comment by skulden13** [comment]
@EncryptMan take a look here https://github.com/nextauthjs/next-auth/discussions/7238
